scripts/e2e-email/validate-email.js

```diff
@@ -40,6 +40,12 @@ var messageContentChecks = [
     xheaders: [],
   },
   {
+    subject: 'New sign-in to Sync',
```

Comment:
Yeah, the message is "if you did not authorize this action, then please change your password.", so the link with new sign-in is `/settings/change_password`.